runtime/src/test_common.rs

```diff
@@ -229,3 +240,10 @@ pub fn sign(payload: &crate::StateChange, keypair: &sr25519::Pair) -> DidSignatu
         value: keypair.sign(&payload.encode()).0,
     })
 }
+
+/// create a random byte array with set len
+pub fn random_bytes(len: usize) -> Vec<u8> {
```

Comment:
I would rather let the caller (who needs random bytes) use the line `(0..len).map(|_| rand::random()).collect()` because he might not always require a `thread_rng`, eg. random number gen with specific seed.
Regarding the `assert`, what is the reason for that. `thread_rng` (relying on `OsRng`) will block in case the seed data is not available but that isn't look practical given we are using for tests.

Reply:
This is a convenience function. It is only exposed to our unit tests.